src/main/java/school/faang/inversion/Main.java

```diff
@@ -0,0 +1,24 @@
+package school.faang.inversion;
+
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args) {
+        final int[] numbers = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+        final int[] reversedNumbers = reverse(numbers);
+
+        System.out.println(Arrays.toString(numbers));
+        System.out.println(Arrays.toString(reversedNumbers));
+    }
+
+    private static int[] reverse(int[] numbers) {
```

Comment:
Засчитывается, а теперь смотри, как можно сделать, используя только входной массив:
```suggestion
    for (int i = 0; i < array.length / 2; i++) {
        int temp = array[i];
        array[i] = array[array.length - 1 - i];
        array[array.length - 1 - i] = temp;
    }
}
```

Reply:
это решение немного некрасивое и надо вчитываться. Моё всё симпатичней